tests/pos/ghost-pathdep-1.scala

```diff
@@ -1,19 +0,0 @@
-object Test {
-
-  fun1(new Bar)
-  fun2(new Bar)
-  fun3(new Bar)
-
-  def fun1[F >: Bar <: Foo](ghost f: F): f.X = null.asInstanceOf[f.X]
-  def fun2[F >: Bar <: Foo](ghost f: F)(ghost bar: f.B): f.B = null.asInstanceOf[f.B]
```

Comment:
Make them neg tests. It is always good to make sure we do not have a regression.